services/quote-aggregator/main.py

```diff
@@ -54,6 +54,10 @@ def loan_quotes(event: CloudEvent):
 
 # app = FastAPI(lifespan=lifespan)
 
+# @app.get("/")
+# async def healthz():
```

Comment:
is this what was alluded in Slack? ie. the health check that prevents subs from lagging?

Reply:
no